webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -89,6 +91,12 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
       </div>,
       label: 'Durations',
     },
+    {
+      content: trial.state === 'ACTIVE' &&
+      `${workloadStatus[0].toUpperCase() + workloadStatus.slice(1)}
+      on batch ${trial.totalBatchesProcessed}`,
+      label: 'Workload Status',
```

Comment:
@ahj-determined any thoughts on what we call this? "Performing: training/checkpointing/validation"

Reply:
( a screenshot would help paint a better picture here

Reply:
Added screenshots and didn't run into any api issues

Reply:
@hamidzr To keep it a noun, what about "Current Workload: Training/Checkpointing/Validation on X"?